.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,89 @@
+name: Bug Report
+description: File a bug report
+labels: ["t/bug"]
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please make sure to add as much detail as you can, preferably with a reproduction. This will help us diagnose the issue faster and thus resolving it quicker.
+        
+        If this is a bug in Xamarin.Forms, please [open the issue in the Xamarin.Forms repo](https://github.com/xamarin/Xamarin.Forms/issues/new?assignees=&labels=s%2Funverified%2C+t%2Fbug+%3Abug%3A&template=bug_report.md&title=[Bug]+).
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Please give us a detailed description of the issue that you're seeing. You can add screenshots, videos and reproduction projects (zipped) here as well.
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+  - type: textarea
+    id: repro-steps
+    attributes:
+      label: Steps to Reproduce
+      description: Describe all the steps we need to take to show the behavior that you have observed.
+      placeholder: | 
+        1. Create a File > New .NET MAUI App
+        2. Add a `Button` like so: `<Button Text="this is a bug" />`
+        3. Click the added button and observe the bug 🐞
+    validations:
+      required: true
+  - type: dropdown
```

Comment:
similar feedback applies to all dropdowns

Reply:
Also, no :( something that is rendered this way by GitHub. If you select a value it will say "Selection: x" where x is the actual selected value.

Did put in a feedback ticket for it: https://github.com/github/feedback/discussions/7532